org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md

```diff
@@ -15,6 +15,18 @@ time=15
 name="Workshop"
 src="blocks/workshop"
 time="140"
+  [[blocks.nested.blocks]]
+    name="TBD [PD] (30 Mins)"
+    src="TBD"
+    time=0
+  [[blocks.nested.blocks]]
+    name="Debugging 3 [Tech] (60 Mins)"
```

Comment:
Debugging React stuff seems quite ambitious? We probably want to remove this one?

Reply:
Where do you think this fits better in the curriculum or should we leave it out completely? 
Also is there a workshop off the top of your head that fits better here?

Reply:
I think we should probably leave it out completely, it relies on material we don't teach at all... Maybe @SallyMcGrath has an idea of a good replacement :)

Reply:
Removed and will pick up this topic over the weekend + open a new pull request

Reply:
https://github.com/CodeYourFuture/CYF-Workshops/tree/main/components-2

https://github.com/CodeYourFuture/CYF-Workshops/tree/main/goblin-breakdown